Docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: rootpassword
```

Comment:
Using 'rootpassword' as the MYSQL_ROOT_PASSWORD is not secure. Consider using a more secure password, especially if this setup is intended for production use.